fli-iam/shanoir-ng
# Fli iam#1698 - Better manage link between subject & animal subject

- Create subject from preclinical back via RabbitMQ
- Use subject id as preclinical id through API
- Display animal subject as such in the tree view
- Synchronize deletion

**How to test :**
- Create a preclinical subject from bruker import
  - Check that it shows in the preclinical subject list
  - Check that it shows in the study tree, as an animal subject
    - Check that clicking details from the node redirect to the preclinical subject view (not the subject view)
  - Check that it does not shows in the subject list
 - Update a preclinical subject
   - Check that the field are well updated
   - Check that an existing subject name can not be created
- Delete a preclinical subject
  - Check that it does not shows anymore in the preclinical subject list
  - Check that it does not shows anymore in the study tree
